internal/usage/usage.go

```diff
@@ -23,6 +23,7 @@ const (
 	DiskSizeGB                      = "Capacity, in gigabytes, of the host’s root volume."
 	Backup                          = "If true, uses Atlas Continuous Backups to back up cluster data."
 	MDBVersion                      = "MongoDB version of the cluster to deploy."
+	AuthDB                          = "Database name."
```

Comment:
I am not sure about the purpose of this new flag, this is why I put this poor description

Reply:
let's use "Authentication database name.